Core/NoOpRunner.Core/ShapesContainer.cs

```diff
@@ -10,7 +10,7 @@ public class ShapesContainer
         [JsonProperty] 
         protected List<BaseShape> shapes { get; set; }
 
-        public int SizeX { get; set; }
+        public int SizeX { get; set; }//can remove? GameSetting have it already
```

Comment:
I guess this is more convenient than taking the value from GameSettings every time?